### PR TITLE
Fix missing deps for bluepysnap

### PR DESCRIPTION
**Pull request by tomdele**
missing pandas deps in bluepysnap package

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-bluepysnap/package.py [hunk 1]
```diff
@@ -25,12 +25,14 @@ class PyBluepysnap(PythonPackage):
     version('0.1.2', sha256='aa29c1344258c1ca0cf7f5f53f3080025b9d098e3366872369586ad7ccf334a2')
 
     depends_on('py-setuptools', type=('build', 'run'))
-
     depends_on('py-functools32', type='run', when='^python@:3.1.99')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-bluepysnap/package.py]: Are we hanging on to this, still?

**Review by matz-e — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-bluepysnap/package.py (hunk 1 above)
Comment (tomdele) [reply var/spack/repos/builtin/packages/py-bluepysnap/package.py]: no I can remove it for sure

**Comment by tomdele** [comment]
@matz-e, I am a bit puzzled about the fail in the tests here.
I am adding a dependency to pandas which is used in bluepy, very common everywhere, I am using old version and it breaks. Is there something I am missing ?
I have no clue what happened.

**Review by matz-e — APPROVED** [review]
@tomdele Seem like some temporary failure in the deployment plan. If this keeps persisting, I'll investigate.

**Comment by tomdele** [comment]
> Seem like some temporary failure in the deployment plan. If this keeps persisting, I'll investigate.

Ok thank you ! I was unsure if it was due to my PR or the previous one or a problem with the plan.

**Comment by tomdele** [comment]
oups sorry for the review re-request that was a misclick.

**Pull request merged**